### PR TITLE
Create Unity Cinder customized container image for RHOSP17.1

### PR DESCRIPTION
**Pull request by wendyzheng0**
Create new cinder image for RHOSP17.1

### CODE REVIEW

File: cinder/Dockerfile [hunk 1]
```diff
@@ -1,35 +1,25 @@
...
 ARG storops_version=1.2.11
 
-LABEL maintainer="Dell EMC" \
-      description="Red Hat OpenStack Platform 16.2 cinder-volume Dell EMC VNX and Unity" \
-      summary="Red Hat OpenStack Platform 16.2 cinder-volume Dell EMC VNX and Unity" \
-      name="dellemc/openstack-cinder-volume-dellemc-rhosp16" \
-      vendor="Dell EMC" \
-      version="4.4.1" \
-      storops=${storops_version} \
-      naviseccli="7.33.9.1.55"
```
Comment (xiedenglu) [inline cinder/Dockerfile:15]: Is this not needed?

Comment (wendyzheng0) [reply cinder/Dockerfile:15]: The RHOSP17 is based on rhel9, the so libs naviseccli needs are for rhel8. Considering we don't have resource to check and make naviseccli work on rhel9 and the customer's requirement is Unity. So drop it first unless there is requirement.

File: cinder/Changelist.md [hunk 2]
```diff
@@ -1,4 +1,3 @@
 | Tags | Image version | Changes | Build date | Build type |
 | ---- | ------------- | ------- | ---------- | ---------- |
-| 16.2.0-6 | 4.4.0 | Initial version of RHOSP16.2.0, base image 16.2.0-61, storops 1.2.7 | 2021-09-17 | Manual |
```
Comment (hanc8) [inline cinder/Changelist.md:3]: Why delete the old history here?

Comment (wendyzheng0) [reply cinder/Changelist.md:3]: This is the first version for RHOSP17, so remove entries for 16. I'm following the practice in previous versions. You can refer to branches for other version.

**Pull request merged**